sessions/TypeScript-100/101/101.001/app.ts

```diff
@@ -0,0 +1,5 @@
+function hello(name: string) {
+    console.log(`Hello, ${name}!`);
+}
+
+hello("World");
```

Comment:
Would be nicer to show the possibility of setting the output type:

```suggestion
function hello(name: string): string {
    return `Hello, ${name}!`;
}

console.log(hello("World"));
```